galasa-simbank-tests/dev.galasa.simbank.tests/src/main/java/dev/galasa/simbank/tests/WebAppIntegrationTest.java

```diff
@@ -0,0 +1,148 @@
+import dev.galasa.zosbatch.IZosBatch;
+import dev.galasa.zosbatch.IZosBatchJob;
+import dev.galasa.zosbatch.IZosBatchJobname;
+import dev.galasa.zosbatch.ZosBatch;
+import dev.galasa.zosbatch.ZosBatchException;
+import dev.galasa.zosbatch.ZosBatchJobname;
+import dev.galasa.selenium.ISeleniumManager;
+import dev.galasa.selenium.IWebPage;
+import dev.galasa.selenium.SeleniumManager;
+import dev.galasa.selenium.SeleniumManagerException;
+import dev.galasa.docker.DockerContainer;
+import dev.galasa.docker.DockerManagerException;
+import dev.galasa.docker.IDockerContainer;
+
+@Test
+public class WebAppIntegrationTest {
+
+	@DockerContainer(image = "maven-web-app", dockerContainerTag = "a", start = true)
+    public IDockerContainer container;
+	
+	@ZosImage(imageTag = "SIMBANK")
+	public IZosImage        image;
+
+	@Zos3270Terminal(imageTag = "SIMBANK")
+	public ITerminal        terminal;
+	
+	@ZosBatch(imageTag="SIMBANK")
+    public IZosBatch zosBatch;
+    
+    @ZosBatchJobname(imageTag="SIMBANK")
+	public IZosBatchJobname zosBatchJobname;
+	
+	@CoreManager
+    public ICoreManager     coreManager;
+	
+	@BundleResources
+	public IBundleResources resources; 
+
+	@SeleniumManager
+    public ISeleniumManager seleniumManager;
+
+	@Logger
+	public Log logger;
+	
+	public static int account;
+	
+	@Test
+	public void webAppIntegrationTest() throws SeleniumManagerException, DockerManagerException, TimeoutException, KeyboardLockedException, TerminalInterruptedException, TextNotFoundException, FieldNotFoundException, NetworkException, TestBundleResourceException, IOException, ZosBatchException{
+
+		//Dianas code
+		terminal.waitForKeyboard()
+		.positionCursorToFieldContaining("Userid").tab().type("IBMUSER")
+		.positionCursorToFieldContaining("Password").tab().type("SYS1")
+		.enter().waitForKeyboard()
+
+        // Open banking application
+		.pf1().waitForKeyboard()
+		.clear().waitForKeyboard().type("bank")
+		.enter().waitForKeyboard();
+
+		String accountNumber = String.valueOf((int)Math.random() * (999999999 - 100000000 + 1) + 100000000);
+
+		terminal.pf1().waitForKeyboard()
+		.positionCursorToFieldContaining("Account Number").tab().type(accountNumber)
+		.enter().waitForKeyboard();
+		
+		
+		while(!terminal.retrieveScreen().contains("Account Not Found")){
+			
+		accountNumber = String.valueOf((int)Math.random() * (999999999 - 100000000 + 1) + 100000000);
+
+		terminal.pf3().pf1().waitForKeyboard()
+		.positionCursorToFieldContaining("Account Number").tab().type(accountNumber)
+		.enter().waitForKeyboard();
+		}
+		
+		HashMap<String, Object> parameters = new HashMap<>();
+        parameters.put("CONTROL", "ACCOUNT_OPEN");
+		parameters.put("DATAIN", accountNumber+",00-00-00,1");	
+		String jcl = resources.retrieveSkeletonFileAsString("/resources/skeletons/SIMBANK.jcl", parameters);
+		IZosBatchJob batchJob = zosBatch.submitJob(jcl, zosBatchJobname);
+		logger.info("batchJob.toString() = " +  batchJob.toString());
```

Comment:
As long as the batch manager logs the JCL response to the RAS I don't think we need to log it